report/5.Solution/MI_solution/object_fill_algorithm.tex

```diff
@@ -1,4 +1,32 @@
-\subsection{Object fill algorithm}\label{solution:objfillalgo}
-% Description of the Object Fill algorithm
-% The point of this would be to describe the journey we went on mainly.
-% 
+\subsection{Object Fill algorithm}\label{solution:objfillalgo}
```

Comment:
Maybe add:
The algorithm was already explained in /section(whatever). As such they will not be explained again.